lib/mpi/responses/parser_base.rb

```diff
@@ -40,25 +40,12 @@ def sanitize_edipi(edipi)
         sanitized_result
       end
 
-      def sanitize_participant_id(participant_id)
-        return if participant_id.nil?
+      def sanitize_id(id)
+        return if id.nil?
 
         # Get rid of non-digit characters like 'UNK'/'ASKU'
-        sanitized_result = participant_id.match(/\d+/)&.to_s
-        if sanitized_result != participant_id
-          Rails.logger.info "Participant id sanitized, was: '#{participant_id}' now: '#{sanitized_result}'."
```

Comment:
this should be modified as per the statement below about not logging sensitive data.